Mathlib/Algebra/Order/Hom/Monoid.lean

```diff
@@ -130,8 +130,8 @@ def OrderMonoidHomClass.toOrderMonoidHom [OrderHomClass F α β] [MonoidHomClass
 
 /-- Any type satisfying `OrderMonoidHomClass` can be cast into `OrderMonoidHom` via
   `OrderMonoidHomClass.toOrderMonoidHom`. -/
-@[to_additive /-- Any type satisfying `OrderAddMonoidHomClass` can be cast into `OrderAddMonoidHom` via
-  `OrderAddMonoidHomClass.toOrderAddMonoidHom` -/]
+@[to_additive /-- Any type satisfying `OrderAddMonoidHomClass` can be cast into `OrderAddMonoidHom`
+  via `OrderAddMonoidHomClass.toOrderAddMonoidHom`. -/]
```

Comment:
Mathlib style is to have subsequent lines in doc-strings be not indented. (This could be fixed in follow-up PRs, though.)
Same below

Reply:
Ah, okay. I attempted to preserve whatever indentation was already present at the time that I initially made these commits. I think I can automate the desired change (to some extent) in a subsequent PR. :)

Reply:
Automatic follow-ups are definitely welcome :-)